minkindr/include/kindr/minimal/rotation-quaternion.h

```diff
@@ -156,9 +156,18 @@ class RotationQuaternionTemplate {
     return vector() == rhs.vector();
   }
 
+  /// \brief Check the validity of a rotation matrix.
+  static bool isValidRotationMatrix(const RotationMatrix& matrix);
+
+  /// \brief Factory to construct a RotationQuaternionTemplate from a near
+  ///        orthonormal rotation matrix.
+  inline static RotationQuaternionTemplate<Scalar> constructAndRenormalize(
+      const RotationMatrix& R) {
+    return RotationQuaternionTemplate<Scalar>(Implementation(R).normalized());
```

Comment:
@HannesSommer, let's add an issue to Kindr that we should support these "Projection" operations somehow. This was meant as a quick-and-dirty fix so that Thomas isn't blocked.